department-of-veterans-affairs/va.gov-cms
# VACMS-12612: Prevent Cypress failing tests on uncaught application exceptions.

## Description

Closes #12612.  Uncaught application exceptions should already be logged and reported upstream by Sentry.  Failing whenever an AJAX call fails randomly is frustrating and wastes time and resources.

## QA steps

See [here](https://docs.cypress.io/api/events/catalog-of-events#To-turn-off-all-uncaught-exception-handling).  This is basically a direct copypasta from upstream documentation.
